coders/heic.c

```diff
@@ -331,8 +285,21 @@ static Image *ReadHEICImage(const ImageInfo *image_info,
     Copy HEIF image into ImageMagick data structures
   */
   (void) SetImageColorspace(image,YCbCrColorspace,exception);
+  decode_options=(struct heif_decoding_options *) NULL;
+  option=GetImageOption(image_info,"heic:preserve-orientation");
+  if (IsStringTrue(option) == MagickTrue)
+    {
+      decode_options=heif_decoding_options_alloc();
+      decode_options->ignore_transformations=1;
+    }
+  else
+    {
```

Comment:
Almost there 👍 Thanks for making all these changes for us. 

We don't use braces when the statement is just one line so could you remove them here?

Reply:
ok

Reply:
done